### ISSUE TITLE
[SF]Icon: A11y - Icon Cannot Be Activated With Enter/Space Key in JAWS/NVDA

### ISSUE THREAD

**Issue by VidhiNPatel** [SAP/ui5-webcomponents#11454]
### Describe the bug

The Icon Cannot Be Activated With the Enter and Space key in "Interactive" mode.
Keyboard Handling is not working in JAWS/ NVDA screen reader
[Space] / [Enter] or [Return] - Fires the click event if the mode property is set to Interactive.
https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-icon--docs 

### Reproduction steps

1. Go to https://stackblitz.com/edit/github-t5j5mn-u4rgcw6c?file=src%2FApp.tsx,src%2Findex.css,package.json 
2. Start the JAWS/NVDA screen reader
3. Use [Space] / [Enter] or [Return] to click on the Icon
4. Observed that the Icon Cannot Be Activated With Enter and Space Key

### UI5 Web Components for React Version

2.9.2

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by NakataCode** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

Could you please look over the issue? It is reproducible only in screen reader environments (JAWS/NVDA). While `<ui5-icon mode="Interactive">` correctly fires the `click` event on mouse, Enter, and Space under normal conditions, the `click` event is not triggered when a screen reader is active and the user presses the Space key. Enter works as expected.

I've also created a minimal [reproducible example](https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw...) using the UI5 Web Components Playground.

Additionally, I found [this related issue in the JAWS GitHub repository](https://github.com/FreedomScientific/VFO-standards-support/issues/829), which suggests that JAWS may suppress key events in certain configurations. It might be worth investigating whether a workaround is possible or if this should be documented as a known limitation.

Best Regards,  
Nikola

**Comment by dobrinyonkov** [comment]
Let me know if you face any issues.

Regards,
Dobrin

**Comment by VidhiNPatel** [comment]
Hi @dobrinyonkov ,
The Central Accessibility (A11y) team has denied your proposal and raised a question about why the issue occurs only with the Icon component when the Virtual PC cursor mode is activated. For example, if we look at the Button component, the click event functions properly whether the Virtual PC cursor mode is turned on or off.

Additionally, since the role attribute is set to "button," it is expected that the active click action should work correctly when using a screen reader.

Could you please provide your thoughts on this issue? I'm reopening this issue

**Comment by VidhiNPatel** [comment]
Hi @Lukas742 and @dobrinyonkov ,

Could you please provide updates on this issue? Thank you!

**Comment by judith9209** [comment]
Will bring this ticket up in next UI5 planning meeting on 29 Jan. 
When JAWS or NVDA is switched on [_onkeydown ](https://github.com/UI5/webcomponents/blob/main/packages/main/src/Icon.ts#L227) is not triggered. Please consider using click event; similar like for buttons. _onkeydown is also incorrect because you cannot cancel this event.

**Comment by dobrinyonkov** [comment]
Hi @judith9209 and @VidhiNPatel, I will research and test this "click event only" suggestion today, will post any findings later.

Thanks and Best Regards,
Dobrin
